packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -71,10 +74,14 @@ export const OnboardingProgressStepper = () => {
                   onSelect={() => showStep(step)}
                   description={step.description}
                   completed={isCompleted}
+                  readOnly={step.readonly}
                   endAdornment={
-                    <Icon name="chevronRight" size="small" color="grey11" />
+                    step.readonly !== true && (
```

Comment:
```suggestion
                    !step.readonly && (
```

Reply:
I made it like that to satisfy `strict-boolean-expressions` ts rule

Reply:
Weren't we gonna disabled this rule? 👀

Reply:
yeah, please 🙏